android/tangram/jni/platform_android.cpp

```diff
@@ -328,5 +337,11 @@ void initGLExtensions() {
     glGenVertexArraysOESEXT = (PFNGLGENVERTEXARRAYSOESPROC) dlsym(libhandle, "glGenVertexArraysOES");
 }
 
+std::string stringFromJString(JNIEnv* jniEnv, jstring string) {
+    const char* cstring = jniEnv->GetStringUTFChars(string, NULL);
```

Comment:
I prefer using `GetStringUTFRegion`. Threat of not releasing the resource is not there.

Refer: https://github.com/tangrams/tangram-es/blob/jni-revisited/android/tangram/jni/platform_android.cpp#L131